pkgs/stdenv/generic/setup.sh

```diff
@@ -422,50 +421,55 @@ fi
 
 
 substitute() {
-    local input="$1"
-    local output="$2"
+    local input=$1
+    local output=$2
```

Comment:
I don't find this an improvement, it's just forces people to learn another special case in bash syntax instead of using the obviously safe variant.